docs/versioning_integration.md

```diff
@@ -73,6 +78,9 @@ class BlogCMSConfig(CMSAppConfig):
             preview_url=get_preview_url
         ),
     ]
+    versioning_add_to_confirmation_context = OrderDict({
+        'unpublish': {'cat_stories': stories_about_intelligent_cats},
```

Comment:
This should be the other way round. It's the inner dict that needs to be ordered.